app/src/theme/index.js

```diff
@@ -378,7 +378,7 @@ const theme = {
   fixedHeightModal: {
     content: {
       height: '510px',
```

Comment:
Don't know how did this height get in there but it's causing issues with your connection modal...Because you have scroll just ommiting it or setting it to auto will fix it....
![Screenshot 2021-05-12 at 12 00 03](https://user-images.githubusercontent.com/33226956/117957486-6b8f3100-b31a-11eb-9823-f3b9ec477503.png)